scalalib/src/mill/javalib/android/AndroidAppModule.scala

```diff
@@ -1051,7 +892,14 @@ trait AndroidAppModule extends JavaModule {
       throw new Exception("Device failed to boot")
   }
 
-  trait AndroidAppTests extends JavaTests {
+  trait AndroidAppTests extends AndroidAppModule with JavaTests {
```

Comment:
maybe try to see if this works with simply AndroidModule as it shouldn't need any app packaging features